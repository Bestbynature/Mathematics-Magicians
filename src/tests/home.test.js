import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/home';

describe('Home', () => {
  it('renders correctly', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it('renders the correct text', () => {
    const { getByText } = render(<Home />);
    expect(getByText('Welcome to our page!')).toBeInTheDocument();
  });
});
