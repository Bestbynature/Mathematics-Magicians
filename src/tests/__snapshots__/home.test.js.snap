// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <div
    class="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      <span>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit animi culpa  
      </span>
      <span>
         nemo, distinctio magni autem reiciendis odio id nihil, voluptatibus recusandae  
      </span>
    </p>
    <p>
      <span>
         Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis nesciunt
      </span>
      <span>
         Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis nesciunt
      </span>
      <span>
         Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis nesciunt
      </span>
    </p>
  </div>
</div>
`;
